test_lottogame.py

```diff
@@ -0,0 +1,107 @@
+import unittest
```

Comment:
In python you must add your tests in a test folder

So in this case you will have a structure like:
-lotto
-tests
-main.py (your entry point)